tools/changelog-generator-wrapper/pnpm-lock.yaml

```diff
@@ -18,15 +18,15 @@ importers:
         specifier: ^0.3.0
```

Comment:
Should these packages still have their own pnpm lock files, now that they're a part of the client release group? Shouldn't `pnpm i` from the root satisfy these?

Reply:
This one can be removed, but the markdown-magic project has a build pipeline, so it needs the local lockfile until we disable/remove the pipeline.

Reply:
Interesting. Not for this PR, but _should_ changelog-generator-wrapper have a pipeline? Even if it's JS, we should validate linting/formatting and run tests.

Reply:
Maybe? With the move to 1ES, pipelines are a lot harder to stand up now (IMO). If we move it to the client release group then it's moot. With the move to 1ES, pipelines are a lot harder to stand up now (IMO).